### PR TITLE
Carthage discontinue release note

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

File: FirebaseCore/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Firebase 8.0.0
 - [removed] The deprecated Firebase InstanceID has been removed. Use Firebase Installations to manage
   app instance and use Firebase Messaging to manage FCM registration token instead. (#7970)
+- [changed] The experimental Carthage distribution is temporarily discontinued pending integration
+  with the pending Carthage 0.38.0 release with support for binary xcframeworks. In the meantime,
```
Comment (ryanwilson) [inline FirebaseCore/CHANGELOG.md:5]: Do you want to add a link to the PR for Carthage? https://github.com/Carthage/Carthage/pull/3152 I'm good either way.

**Review by ryanwilson — APPROVED** [review]

File: FirebaseCore/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+  a mix of 7.4.0 and 7.11.0 will be the latest Carthage distribution.
```
Comment (ryanwilson) [inline FirebaseCore/CHANGELOG.md:6]: Additionally, maybe mentioning that zip files are also a reasonable workaround?

**Comment by agirault** [comment]
FYI, Carthage 0.38.0 was released: https://github.com/Carthage/Carthage/pull/3152

**Comment by paulb777** [comment]
Thanks @agirault.  It's happened too late to make the 8.0.0 release. We'll track adding Carthage back in #7039

**Pull request merged**